modules/@apostrophecms/i18n/i18n/en.json

```
@@ -243,6 +243,7 @@
   "localeSwitcherDiscardChangesAffirmative": "Save as draft and switch",
   "localeSwitcherDiscardChangesNegative": "Discard changes and switch",
   "localeSwitcherDiscardChangesPrompt": "Do you want to discard changes to this {{ docType }} before switching locales?",
+  "localeSwitcherPermissionToCreate": "You don't have permission to create a new {{ docType }} in this locale",
```

```suggestion
  "localeSwitcherPermissionToCreate": "You do not have permission to create a new {{ docType }} in this locale",
```

a little nitpick but seems like we use the "do not" formal form in this file